app/src/main/java/com/example/ithaca_transit_android_v2/Models/Route.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.ithaca_transit_android_v2.Models
+import com.example.ithaca_transit_android_v2.models.Coordinate
+import com.example.ithaca_transit_android_v2.models.Location
+import java.util.*
```

Comment:
please don't use the general `java.util.*` import; you can simplify it to just the classes that you need to import from `java.util`